src/edu/csus/ecs/pc2/core/util/CLICSVerionInfo.java

```diff
@@ -0,0 +1,60 @@
+package edu.csus.ecs.pc2.core.util;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.VersionInfo;
+import edu.csus.ecs.pc2.services.core.JSONUtilities;
+
+/**
+ * CLICS Version Info.
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public class CLICSVerionInfo {
+
+    public CLICSVerionInfo(VersionInfo versionInfo) {
+        version = versionInfo.getPC2Version() + " build " + versionInfo.getBuildNumber();
```

Comment:
According to the spec, the version should be the version of the API not the version of the provider.  In this case "2022-07".  Having just gone through this with my "feed" simulator...
I believe the "name" property is what we should set to be "PC2v9 build xxxxxx", etc.  "name" is the provider information.

Reply:
pushed fix.   The example showed name is kattis, so now name shows pc2
added ccs_version with full pc2 version, ex. 9.9build 20230403 (Monday, April 3rd 2023 18:10 UTC) build 6402

Ex output:
{"ccs_version":"9.9build 20230403 (Monday, April 3rd 2023 18:10 UTC) build 6402","version":"2022-07","version_url":"[https://ccs-specs.icpc.io/2022-07/contest_api","name":"pc2"}](https://ccs-specs.icpc.io/2022-07/contest_api%22,%22name%22:%22pc2%22%7D)